### PR TITLE
gui-apps/wayvnc: fast and simple vnc server for wayland

### PR DESCRIPTION
**Pull request by epsilon-0**
One of the first VNC servers for wayland supporting wlroots based compositors
A big step in getting a wayland only desktop environment and pretty useful for the wayland ecosystem.

### CODE REVIEW

**Comment by epsilon-0** [comment]
@flappyports whenever you get time. Thanks a lot for all your work!

File: gui-libs/neatvnc/neatvnc-0.2.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+LICENSE="ISC"
+SLOT="0"
+IUSE="examples ssl jpeg tracing"
+
+DEPEND="
+	x11-libs/pixman
+	x11-libs/libdrm
+	dev-libs/aml
+	sys-libs/zlib
+	ssl? ( net-libs/gnutls )
```
Comment (Freed0mBear) [inline gui-libs/neatvnc/neatvnc-0.2.0.ebuild:29]: Does the package only support gnutls?

Comment (epsilon-0) [reply gui-libs/neatvnc/neatvnc-0.2.0.ebuild:29]: unfortunately yes.
seems to be a hard requirement.

Comment (epsilon-0) [reply gui-libs/neatvnc/neatvnc-0.2.0.ebuild:29]: as seen here:  https://github.com/any1/neatvnc/blob/master/meson.build#L45

I'm am a bit sad as well given recent security problems with gnutls

**Comment by epsilon-0** [comment]
@Freed0mBear a small reminder about this.

File: dev-libs/aml/aml-0.1.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="event timer executor loop"
+
```
Comment (SoapGentoo) [inline dev-libs/aml/aml-0.1.0.ebuild:9]: remove empty line

File: dev-libs/aml/aml-9999.ebuild [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="event timer executor loop"
+
```
Comment (SoapGentoo) [inline dev-libs/aml/aml-9999.ebuild:9]: ditto

File: gui-libs/neatvnc/files/neatvnc-0.2.0-systemtap_optional.patch [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+@@ -87,7 +87,7 @@ if gnutls.found()
+ 	config.set('ENABLE_TLS', true)
+ endif
+ 
+-if host_system == 'linux' and cc.has_header('sys/sdt.h')
++if host_system == 'linux' and get_option('systemtap') and cc.has_header('sys/sdt.h')
+ 	config.set('HAVE_USDT', true)
+ endif
+ 
+diff --git a/meson_options.txt b/meson_options.txt
```
Comment (SoapGentoo) [inline gui-libs/neatvnc/files/neatvnc-0.2.0-systemtap_optional.patch:14]: is this upstreamed?

File: gui-libs/neatvnc/neatvnc-0.2.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="liberally licensed VNC server library with a clean interface"
+
```
Comment (SoapGentoo) [inline gui-libs/neatvnc/neatvnc-0.2.0.ebuild:9]: empty line

File: gui-libs/neatvnc/neatvnc-0.2.0.ebuild (hunk 1 above)
Comment (SoapGentoo) [reply gui-libs/neatvnc/neatvnc-0.2.0.ebuild:29]: also, add `:=` to gnutls and `libjpeg-turbo`

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: dev-libs/aml/aml-0.1.0.ebuild (hunk 2 above)
Comment (epsilon-0) [reply dev-libs/aml/aml-0.1.0.ebuild]: done

File: dev-libs/aml/aml-9999.ebuild (hunk 3 above)
Comment (epsilon-0) [reply dev-libs/aml/aml-9999.ebuild]: done

File: gui-libs/neatvnc/files/neatvnc-0.2.0-systemtap_optional.patch (hunk 4 above)
Comment (epsilon-0) [reply gui-libs/neatvnc/files/neatvnc-0.2.0-systemtap_optional.patch:14]: yes, i've removed it from 999

File: gui-libs/neatvnc/neatvnc-0.2.0.ebuild (hunk 5 above)
Comment (epsilon-0) [reply gui-libs/neatvnc/neatvnc-0.2.0.ebuild]: done

File: gui-libs/neatvnc/neatvnc-0.2.0.ebuild (hunk 1 above)
Comment (epsilon-0) [reply gui-libs/neatvnc/neatvnc-0.2.0.ebuild]: done

**Pull request closed without merging**